packages/stream-tests/src/__tests__/node-status.test.ts

```diff
@@ -93,40 +78,4 @@ describe('Node Status Endpoint tests', () => {
       expect(typeof addr).toEqual('string')
     }
   })
-
-  test('Pending anchor counter test', async () => {
```

Comment:
does this change break the `pendingAnchors` reporting in the `nodeStatus` endpoint?  If so, should we remove that field from the nodeStatus output rather than just have it always return 0?

Reply:
There is a ticket to handle it separately. This PR would be tooo big to review if `nodeStatus` is also taken care of.